hazelcast/src/main/java/com/hazelcast/client/impl/management/ManagementCenterService.java

```diff
@@ -335,4 +337,63 @@ public void shutdownMember(Member member) {
                         MCGetTimedMemberStateCodec.decodeResponse(clientMessage).timedMemberStateJson)
         );
     }
+
+    /**
+     * Checks if local MC config (client B/W list) on a given member has the same ETag as provided.
+     *
+     * @param member    target member
+     * @param eTag      ETag value of MC config to match with (should be the latest value from MC)
+     * @return          operation future object with match result: <code>true</code> if config ETags match
+     */
+    @Nonnull
+    public CompletableFuture<Boolean> matchMCConfig(Member member, String eTag) {
```

Comment:
Naming is misleading. I understand about changing the whole MC config but all you change is ETag. Can't you use a naming that means this, e.g. `matchEtags`

Reply:
This operation doesn't change anything, i.e. it's a read-only operation. It simply matches config ETags and return the result. The `matchEtags` name doesn't highlight the fact that this operation is all about MC config (which includes only client filtering for now, but may be expanded in future).

See the TDD for more details on operations: https://hazelcast.atlassian.net/wiki/spaces/EN/pages/1767178346/MC+New+Communications+with+Cluster+-+TDD#MC:NewCommunicationswithCluster-TDD-ClientB/WFilterConfigOperations